test/polar/Sunburst.spec.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { SunburstChart } from '../../src';
+
+describe('<Sunburst />', () => {
+  const data = {
+    name: 'Root',
+    value: 100,
+    children: [
+      {
+        name: 'Child1',
+        fill: '#264653',
+        value: 30,
+        children: [
```

Comment:
Could you please simplify the test as much as possible? The test data should be only as complex as need be.